src/utils/wallet.js

```diff
@@ -400,6 +601,14 @@ class Wallet {
         return new nearApiJs.Account(this.connection, accountId)
     }
 
+    async getAccountAndState(accountId) {
```

Comment:
it's best to keep method doing 1 thing unless there is a good reason not to. Not sure what is the reason here

Reply:
needed to get state and check has2fa in multiple places

Reply:
I think it's best to return extended `Account` object instead from `getAccount()`. Then that object can know that it's 2FA and act accordingly when signing/sending transactions.